src/components/Dashboard.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+
+const Dashboard = (props) => {
+    const [reviews, setReviews] = useState([]);
+    useEffect(() => {
+        fetch('https://pacific-citadel-03998.herokuapp.com/api/reviews')
+        .then(res => res.json())
+        .then((data) => {
+            console.log(data);
+            setReviews(data.reviews);
+        })
+    }, []);
+    return (
+    <ul>
+        {reviews.map((item) => (
+            <div class="review-box">
+                <img class="review-image" src={item.review_img_url} alt={item.title}/>
```

Comment:
this should be `className="review-image"`  as `class` is used for other things within React